manifests/m/MSI/Afterburner/4.6.5.Beta4/MSI.Afterburner.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MSI.Afterburner
+PackageVersion: 4.6.5.Beta4
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: nullsoft
+  NestedInstallerFiles:
+  - RelativeFilePath: MSIAfterburnerSetup465Beta4.exe
+  InstallerUrl: https://download-eu2.guru3d.com/afterburner/[Guru3D.com]-MSIAfterburnerSetup465Beta4Build16358.zip
+  InstallerSha256: 87E15ED6C587B7A187D29A73452FC7F581E72FB1634199AA82D8C64F9F6FDF01
```

Comment:
```suggestion
  InstallerUrl: https://download.msi.com/uti_exe/vga/MSIAfterburnerSetup.zip  InstallerSha256: D1C09F505D9483E93DF5126241C209E8C871656493CE511B3238DF73B3EEE71F
```

Reply:
It's not possible to download Afterburner from MSI website because it requires dynamically generated token. Guru3D is an official distributer as well. More info here: https://github.com/microsoft/winget-pkgs/issues/1990